lib/logatron/logatron.rb

```diff
@@ -16,10 +17,10 @@ def logger
       @log ||= Logatron::BasicLogger.new
     end
 
-    def log_exception(e, severity, additional_info = {})
-      # 'additional_info' can be a flat hash or anything with '#to_s'
-      message = exception_message(e, additional_info)
-      logger.send(severity.downcase, message)
+    # @param additional_info [Object] Typically a flat hash, but can be anything that responds to '#to_s'
+    def log_exception(e, severity, additional_info = {}, error_formatter = Logatron::ErrorFormatter.new)
```

Comment:
The `ErrorFormatter` seems like it should be a configuration setting for Logatron in general. Is there usefulness in allowing the error formatting to set when invoking the method as opposed to configuring Logatron at startup?